ooni/ui/web/web.py

```diff
@@ -14,16 +19,23 @@ def __init__(self, director, scheduler, port_number=8842):
         self.port_number = port_number
 
     def startService(self):
+        """Start a web user interface.
+
+        Connects a given protocol factory to the given numeric TCP/IP port and open a page."""
```

Comment:
Given the fact that this is a standard interface for multiservices I think these two comments are redundant.